src/search/cost_test_ebpf.cc

```diff
@@ -16,8 +16,10 @@ double get_error_cost(inst* p1, inst* p2, int win_start, int win_end) {
   static_safety_check_pgm(prog1.inst_list, inst::max_prog_len);
   c.set_orig(&prog1, inst::max_prog_len, win_start, win_end);
   prog_static_state pss;
+  //This is to infer program state while entering and leaving the window.
```

Comment:
It's generally a good idea to split commits into the smallest possible chunks, where each commit does one small and specific thing. This helps us understand the specific changes adding a feature later on.

Reply:
I for one would be happy to review multiple small commits and PRs :)